db/migrations/0002_alter_skill_bonus.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2024-12-15 12:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='skill',
+            name='bonus',
+            field=models.CharField(max_length=255),
```

Comment:
The `bonus` field is altered to `models.CharField(max_length=255)`. If `bonus` is meant to store numeric values, consider using `models.IntegerField()` or `models.DecimalField()` instead of `CharField`.